plugins/woocommerce/client/admin/client/settings-payments/components/payment-settings-section/payment-settings-section.tsx

```diff
@@ -0,0 +1,28 @@
+/**
+ * External dependencies
+ */
+
+/**
+ * Internal dependencies
+ */
+import './payment-settings-section.scss';
+
+export const PaymentSettingsSection = ( {
```

Comment:
```suggestion
export const PaymentGatewaySettingsSection = ( {
```

Reply:
Removed in 370a44a843242f748568e6b9369d4d13cad5559b